### PR TITLE
Merge master to webots r2018a revision 1

### PR DESCRIPTION
**Pull request by fabienrohrer**
Fixed simple conflicts in `camera-sensors.md`, `lidar-sensors.md` and `using-python.md`

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

**Comment by DavidMansolino** [comment]
👀

File: guide/using-python.md [hunk 1]
```diff
@@ -21,15 +17,15 @@ Webots starts Python using the standard `python` command line.
...
-If you set it for example to `python3.6` instead of `python`, this version of python will be used (if available from the command line). Finally, it is also possible to set a different version of Python for each robot controller by editing the `[python]` section of the `runtime.ini` file in each robot controller directory and setting the `COMMAND` value to `python3`, `python3.6` or `python2.7`, etc.
-If specified in the `runtime.ini` file of a controller, this Python command will be executed instead of the default one to launch this controller.
+If you set it for example to `python3.6` instead of `python`, this version of python will be used (if available from the command line).
+Finally, it is also possible to set a different version of Python for each robot controller by editing the `[python]` section of the `runtime.ini` file in each robot controller directory and setting the `COMMAND` value to `python3`, `python3.6` or `python2.7`, etc. If specified in the `runtime.ini` file of a controller, this Python command will be executed instead of the default one to launch this controller.
```
Comment (DavidMansolino) [inline guide/using-python.md]: This one looks wrong (not sure if it is a merge conflicts issue or if it was already the case).

Comment (fabienrohrer) [reply guide/using-python.md]: It is wrong on master:

https://raw.githubusercontent.com/omichel/webots-doc/master/guide/using-python.md

I'm doing a PR targeting master to fix this.

**Comment by fabienrohrer** [comment]
Ok it seems correct now.

👍

**Pull request merged**